src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -143,7 +143,7 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
                 return this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload));
             })
             .catch(error => {
-                if (!(error instanceof RequestError) || !some(error.body.errors, { code: 'three_d_secure_required' })) {
+                if (!(error instanceof RequestError) || (!some(error.body.errors, { code: 'three_d_secure_required' }) && !some(error.body.errors, { code: 'BankTransferRequired' }))) {
```

Comment:
I think error codes from our payment service should be in snake case. I understand Adyen's `createFromAction` method expects `BankTransferRequired` as the action type. So what you should do here instead is to map the error code coming from BigPay to the format Adyen expects.